cmd/deploy/kubeconfig.go

```diff
@@ -84,7 +85,8 @@ func (*KubeConfig) GetCMDAPIConfig() (*clientcmdapi.Config, error) {
 	return okteto.Context().Cfg, nil
 }
 
-//GetTempKubeConfigFile returns a where the temp kubeConfigFile should be stored
+// GetTempKubeConfigFile returns a where the temp kubeConfigFile should be stored
 func GetTempKubeConfigFile(name string) string {
-	return fmt.Sprintf(tempKubeConfigTemplate, config.GetUserHomeDir(), name, time.Now().UnixMilli())
+	tempKubeConfigTemplate := fmt.Sprintf("kubeconfig-%s-%d", name, time.Now().UnixMilli())
```

Comment:
the destroy version of this func is `kubeconfig-destroy....` can we add here then `deploy` to the name?

Reply:
I prefer to remove it if we are not using it, but I mentioned @ifbyol to know if we needed for any other thing

Reply:
I added your suggestion since we need it for both commands 😉